R/queue_didehpc.R

```diff
@@ -347,8 +350,14 @@ submit_dide <- function(obj, data, task_ids, names) {
     writeLines(glue_whisker(batch_template, list(task_id = id)), batch)
     path <- windows_path(file.path(data$paths$remote$batch, base))
     p()
+    if (length(depends_on) == 0L) {
+      deps <- character(0)
+    } else {
+      deps <- obj$dide_id(depends_on[[id]])
+    }
+    deps <- ifelse(length(deps) > 0, paste0(deps, collapse = ","), "")
```

Comment:
The conditional is not needed (also `ifelse` is generally worth avoiding over an in-line `if` / `else`
)